### PR TITLE
refac(core): improved messages

### PR DESCRIPTION
**Pull request by joaoscheuermann**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -51,8 +61,18 @@ dolinhoCommands.set("usd", {
...
+      dolinhoMarketLastRate = dolinhoMarketCurrentRate = await oxr.getLatestRate('BRL')
+    }
+
+    const [marketVariation, marketVariationPercentage] = calculateVariation(dolinhoMarketLastRate ?? 0, dolinhoMarketCurrentRate ?? 0)
+
+    if (marketVariation === 0) {
+      await interaction.reply(`O **USD** ta valendo **${dolinhoMarketCurrentRate} BRL**!`);
+    } else {
+      await interaction.reply(`O **USD** ta valendo **${dolinhoMarketCurrentRate} BRL**! (${ marketVariation > 0 ? "👆🏻" : "👇🏻" } ${marketVariation} | ${marketVariationPercentage}%)`);
+    }
```
Comment (CaioCS97) [inline src/index.ts:75]: nice!

File: src/index.ts [hunk 2]
```diff
@@ -100,22 +120,97 @@ dolinhoClient.on(Events.ClientReady, async () => {
     }
   }
 
-  // Initialize the cron job that send updates to each channel
   // TODO: armazenad valores historicos para adicionar emoji de que o valor subiu ou desceu
```
Comment (CaioCS97) [inline src/index.ts:123]: I think this is done

File: src/utils/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export function calculateVariation (originalValue: number, currentValue: number): [number, number] {
+  const variation = currentValue - originalValue
+  const percentage = (variation / originalValue) * 100
+
+  return [variation, percentage]
+}
```
Comment (CaioCS97) [inline src/utils/index.ts:6]: missing empty line

**Review by CaioCS97 — CHANGES_REQUESTED** [review]
.

File: src/index.ts (hunk 1 above)
Comment (joaoscheuermann) [reply src/index.ts:75]: 🚀

File: src/index.ts (hunk 2 above)
Comment (joaoscheuermann) [reply src/index.ts:123]: removed

**Comment by joaoscheuermann** [comment]
Fixed PR comments!

**Review by CaioCS97 — APPROVED** [review]

**Pull request merged**